apps/heft/src/pluginFramework/InternalHeftSession.ts

```diff
@@ -32,8 +32,6 @@ function* getAllTasks(phases: Iterable<HeftPhase>): Iterable<HeftTask> {
   }
 }
 
-const FORBIDDEN_SOURCE_FILE_GLOBS: string[] = ['package.json', '.gitingore', 'config/**/*', '.rush/**/*'];
```

Comment:
Are we no longer forbidding these?

Reply:
There isn't currently any special watcher logic in Heft to detect changes to them. It's something we could add back, but would require having a code path that expressly listens for changes to said files.